src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form:not(:last-child) {
+  margin-bottom: 20px;
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). This helps to avoid potential margin collapse and keeps the styling consistent.